src/utilities/OpenWeather.test.ts

```diff
@@ -0,0 +1,16 @@
+import getTemperature from './OpenWeather';
+
+describe('OpenWeather', () => {
+  it('returns promise containing value string if valid request', () => {
+    return getTemperature('60606').then(value => {
```

Comment:
In the future, we'll want to mock the request results so that we don't have to make a request to the API every time we are running tests. 

https://stackabuse.com/using-mocks-for-testing-in-javascript-with-jes/

Reply:
Added mocks and restructed tests to use said mocks